charts/k8s-monitoring/templates/agent_config/_pod_logs.river.txt

```diff
@@ -1,9 +1,23 @@
 {{ define "agent.config.logs.pod_logs" }}
 {{- with .Values.logs.pod_logs }}
 // Pod Logs
+discovery.kubernetes "pods" {
+    role = "pod"
```

Comment:
Couple of questions about this line. One is a nit: the indentation isn't quite right. The other is my real question: is it intentional for this line to appear both here and nested under `selectors` below?

Reply:
Yeah, it's odd behavior, but it's correct: https://grafana.com/docs/agent/latest/flow/reference/components/discovery.kubernetes/#limit-to-only-pods-on-the-same-node